internal/controllers/rollout/controller.go

```diff
@@ -94,17 +95,6 @@ func (c *controller) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	return ctrl.Result{}, nil
 }
 
-func SwapStates(comp *apiv1.Composition) {
```

Comment:
I missed one implementation of swapStates when refactoring the other day - now we're down to one implementation in the synthesis controller.